src/main/java/edu/kis/vh/nursery/StackInterface.java

```diff
@@ -1,11 +1,15 @@
 package edu.kis.vh.nursery;
 
 public interface StackInterface {
```

Comment:
Names of operations do not correspond to operating the stack, thus do not reflect the interface name.